src/commands/debugger/logs/list.js

```diff
@@ -75,10 +76,25 @@ class DebuggerLogsList extends TwilioClientCommand {
 
   outputLogEvents(logEvents) {
     if (logEvents.length > 0) {
+      // Provide a human readable error message
+      logEvents = logEvents.map(e => {
+        return Object.assign({}, e, {
+          alertText: this.formatAlertText(e.alertText)
+        });
+      });
       this.output(logEvents, this.flags.properties, { showHeaders: this.showHeaders });
       this.showHeaders = false;
     }
   }
+
+  formatAlertText(text) {
+    try {
+      const data = querystring.parse(text);
+      return data.Msg || text;
```

Comment:
This looks more representative to me: `return data.parserMessage || data.Msg || text;`

Reply:
Nice! Put in a PR here too: https://github.com/twilio-labs/plugin-watch/pull/1